### ISSUE TITLE
Consistent API Across Pytorch and Tensorflow?

### ISSUE THREAD

**Issue by bhairavmehta95** [awslabs/amazon-sagemaker-examples#473]
Is there any way to do Tensorflow Extending of Containers, like there is for Pytorch? For example - this example seems less intuitive than the Pytorch code, where I can just specify `SAGEMAKER_PROGRAM`.

https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/tensorflow_bring_your_own/container/cifar10/train

**Comment by laurenyu** [comment]
@bhairavmehta95 we [just released](https://github.com/aws/sagemaker-python-sdk/pull/475) TF containers that use the same interface as the PyTorch containers. Documentation is on its way out, but you can take advantage of this by setting `py_version='py3'` or `script_mode=True` when creating a `TensorFlow` estimator instance:

```python
estimator = TensorFlow(entry_script, role, ..., py_version='py3', image_name='your-custom-image-uri')
```

**Comment by bhairavmehta95** [comment]
perfect - exactly what I need! Thanks a lot!
